### DISCUSSION TITLE
Remove the home page

### DISCUSSION THREAD

**Discussion by spryzone** [fuma-nama/fumadocs#369] [Q&A] [answered]
How can I remove the home page and promote the docs up to the root level, i.e. a documentation only site with no landing page?

**Comment by fuma-nama** [comment]
Yes, you can change the routing structure, remember to change your `baseUrl` as well in the source file (`source.ts`)

**Reply by spryzone** [reply]
Thanks, I wrote my changes below in case someone else needs to do this.

**Comment by spryzone** [comment, marked as answer]
Example site: [https://github.com/austererisk/austererisk-com](https://github.com/austererisk/austererisk-com)

Procedure:

1. Move content from `/content/docs` to `/content`
2. Delete .map.ts
3. Merge the layouts by placing the DocsLayout inside the Layout, then delete the `app/docs' folder
4. Modify the `app/source.ts` file as below

app/layout.tsx
```tsx
import { RootProvider } from "fumadocs-ui/provider";
import "fumadocs-ui/style.css";
import { Inter } from "next/font/google";
import type { ReactNode } from "react";
import { DocsLayout } from "fumadocs-ui/layout";
import { pageTree } from "./source";
import Image from "next/image";
import './styles.css'

const inter = Inter({
	subsets: ["latin"],
});

export default function Layout({ children }: { children: ReactNode }) {
	return (
		<html lang="en" className={inter.className}>
			<body>
				<RootProvider>
					<DocsLayout tree={pageTree} nav={{ title: 'My Site' }}>
						{children}
					</DocsLayout>
				</RootProvider>
			</body>
		</html>
	);
}
```

app/source.ts
```ts
import { map } from '@/.map';
import { createMDXSource } from 'fumadocs-mdx';
import { loader } from 'fumadocs-core/source';

export const { getPage, getPages, pageTree } = loader({
  baseUrl: '',
  rootDir: '',
  source: createMDXSource(map),
});
```
